src/techjoomla/tjnotifications/tjnotifications.php

```diff
@@ -1,3 +1,6 @@
+use Joomla\CMS\MVC\Model\BaseDatabaseModel;
```

Comment:
this needs to be after opening php tags

Reply:
probably bcoz this file does not have

defined('_JEXEC') or die();

Reply:
done